packages/compass-collection-stats/src/components/document-stats-item/document-stats-item.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { css } from '@mongodb-js/compass-components';
+import { uiColors } from '@leafygreen-ui/palette';
+
+import CollectionStatsItem from '../collection-stats-item';
+
+const documentStatsItemStyles = css({
+  paddingLeft: '15px',
+  paddingRight: '3px',
+  borderRight: `1px solid ${uiColors.gray.light2}`,
+  flexBasis: 'auto',
+  flexGrow: 0,
+  flexShrink: 0,
+  display: 'flex',
+  alignItems: 'flex-end',
+  marginBottom: 0,
+  '&:last-child': {
+    borderRight: 'none',
+  },
+});
+
+type DocumentStatsItemProps = {
+  documentCount: string;
+  isTimeSeries: boolean;
+  storageSize: string;
+  avgDocumentSize: string;
+};
+
+/**
+ * The document stats item component.
+ */
+const DocumentStatsItem: React.FunctionComponent<any> = ({
```

Comment:
Similar comment to above, we might be able to reduce `any` usage here?
```suggestion
const DocumentStatsItem: React.FunctionComponent<DocumentStatsItemProps> = ({
```